pubspec.yaml

```diff
@@ -1,16 +1,16 @@
 name: code_transformers
 version: 0.2.9
-author: "Dart Team <misc@dartlang.org>"
+author: Dart Team <misc@dartlang.org>
 description: Collection of utilities related to creating barback transformers.
 homepage: https://github.com/dart-lang/code-transformers
+environment:
+  sdk: '>=1.0.0 <2.0.0'
 dependencies:
-  analyzer: ">=0.15.6 <0.26.0"
-  barback: ">=0.14.2 <0.16.0"
-  cli_util: ">=0.0.1 <0.1.0"
-  path: ">=0.9.0 <2.0.0"
-  source_maps: ">=0.9.4 <0.11.0"
-  source_span: ">=1.0.0 <2.0.0"
+  analyzer: '>=0.25.0 <0.27.0'
```

Comment:
can you actually leave the lower bound at 0.15.6 for this package, unless there was an issue that came up?